### ISSUE TITLE
Unable to analyze candle (OHLCV) data for pair BNX/USDT: 'BNX/USDT'

### ISSUE THREAD

**Issue by tomjrtsmith** [freqtrade/freqtrade#7811]
## Describe your environment

  * Operating system: ____Ubuntu 22.04
  * Python Version: _____ (`python -V`) 3.10.4
  * CCXT version: _____ (`pip freeze | grep ccxt`) 2.1.96
  * Freqtrade Version: ____ (`freqtrade -V` or `docker-compose run --rm freqtrade -V` for Freqtrade running in docker) freqtrade 2022.12.dev-320535a22
  
Note: All issues other than enhancement requests will be closed without further comment if the above template is deleted or not filled out.

## Describe the problem:

*Explain the problem you have encountered* 

This is happening on every Freqai strat that I run - four so far - and this is on the freqai_rl_test_strat. 

freqtrade trade --config config.rl.json --strategy freqai_rl_test_strat --freqaimodel ReinforcementLearner

### Steps to reproduce:

  1. _____
  2. _____
  3. _____
  
### Observed Results:

  * What happened?
  * What did you expect to happen?

### Relevant code exceptions or logs

Note: Please copy/paste text of the messages, no screenshots of logs please.

  ```
2022-11-28 01:50:24,600 - freqtrade.strategy.interface - WARNING - Unable to analyze candle (OHLCV) data for pair BNX/USDT: 'BNX/USDT'
2022-11-28 01:50:24,602 - freqtrade.strategy.strategy_wrapper - ERROR - Unexpected error 'EOS/USDT' calling <bound method IStrategy._analyze_ticker_internal of <freqai_rl_test_strat.freqai_rl_test_strat object at 0xffff7c28c520>>
Traceback (most recent call last):
  File "/home/ubuntu/freqtrade/freqtrade/strategy/strategy_wrapper.py", line 27, in wrapper
    return f(*args, **kwargs)
  File "/home/ubuntu/freqtrade/freqtrade/strategy/interface.py", line 748, in _analyze_ticker_internal
    dataframe = self.analyze_ticker(dataframe, metadata)
  File "/home/ubuntu/freqtrade/freqtrade/strategy/interface.py", line 726, in analyze_ticker
    dataframe = self.advise_indicators(dataframe, metadata)
  File "/home/ubuntu/freqtrade/freqtrade/strategy/interface.py", line 1221, in advise_indicators
    return self.populate_indicators(dataframe, metadata)
  File "/home/ubuntu/freqtrade/user_data/strategies/freqai_rl_test_strat.py", line 74, in populate_indicators
    dataframe = self.freqai.start(dataframe, metadata, self)
  File "/home/ubuntu/freqtrade/freqtrade/freqai/freqai_interface.py", line 138, in start
    dk = self.start_live(dataframe, metadata, strategy, self.dk)
  File "/home/ubuntu/freqtrade/freqtrade/freqai/freqai_interface.py", line 356, in start_live
    self.dd.update_historic_data(strategy, dk)
  File "/home/ubuntu/freqtrade/freqtrade/freqai/data_drawer.py", line 648, in update_historic_data
    self.current_candle = history_data[dk.pair][self.config['timeframe']].iloc[-1]['date']
KeyError: 'EOS/USDT'
  ```

**Comment by robcaulk** [comment]
It looks like you are probably trying to use a dynamic pairlist. In FreqAI you are only allowed to use a constant list (but you can rearrange the order with e.g. shuffle or volume).

We have documented this in “common pitfalls” part of the documentation:

https://www.freqtrade.io/en/stable/freqai/#common-pitfalls

**Comment by tomjrtsmith** [comment]
Ah that is it. Thankyou.

**Comment by tomjrtsmith** [comment]
Maybe a bit premature, I edited the config and replaced the dynamic pairlist with a static one, basically the same setup carried over from a docker install. I ran @lolongcovas binary classifier strat and it failed and then I ran the FreqaiExampleStrat which I was sure would work and it failed. I'm looking over my system to see what may be causing it, happy to post configs and the offending strat files. One thing I've noticed and I don't know how or why this is the case but I'm running this from a vps and there are no files in the freqaimodels folder which is different from the docker setup which automatically referenced them so have to copy and paste, but I don't think this is causing the issue.
